### PR TITLE
fix(ghrunner): fix release pipeline for self-hosted runner

### PR DESCRIPTION
**Pull request by tjololo**
## Description
* Altinn has uppercase, thats not allowed in ghcr.io 
* Remove ghrunner- prefix from image tag

### CODE REVIEW

File: .github/workflows/gh-runner-release.yml [hunk 1]
```diff
@@ -38,6 +39,6 @@ jobs:
           context: ./infrastructure/images/gh-runner
           push: true
           platforms: linux/amd64,linux/arm64
-          tags: ghcr.io/${{ github.repository }}/gh-runner:v${{ steps.version.outputs.version }}
+          tags: ghcr.io/${{ stepd.vars.outputs.reponame }}/gh-runner:v${{ steps.vars.outputs.version }}
```
Comment (monteiro-renato) [inline .github/workflows/gh-runner-release.yml:42]:
```suggestion
          tags: ghcr.io/${{ steps.vars.outputs.reponame }}/gh-runner:v${{ steps.vars.outputs.version }}
```

File: .github/workflows/gh-runner-release.yml [hunk 2]
```diff
@@ -38,6 +39,6 @@ jobs:
...
         env:
           SOURCE_DATE_EPOCH: ${{ env.TIMESTAMP }}
```
Comment (monteiro-renato) [inline .github/workflows/gh-runner-release.yml:44]:
```suggestion
          SOURCE_DATE_EPOCH: ${{ env.TIMESTAMP }}
```

File: .github/workflows/gh-runner-release.yml (hunk 1 above)
Comment (tjololo) [reply .github/workflows/gh-runner-release.yml:42]: Nice catch! Thnx

File: .github/workflows/gh-runner-release.yml (hunk 2 above)
Comment (tjololo) [reply .github/workflows/gh-runner-release.yml:44]: "Typo" while doing review on phone?

Comment (monteiro-renato) [reply .github/workflows/gh-runner-release.yml:44]: New line at the end of the file :) It's just a nit.

**Review by monteiro-renato — APPROVED** [review]

File: .github/workflows/gh-runner-release.yml (hunk 2 above)
Comment (tjololo) [reply .github/workflows/gh-runner-release.yml:44]: I like nits!

**Pull request merged**